image/spec/README.md

```diff
@@ -0,0 +1,47 @@
+# Docker Image Specification v1.
+
+This directory contains documents about Docker Image Specification v1.X.
+
+Distribution and archival specifications such as file layouts and manifests are no longer used
+in Moby and Docker, except in `docker save` and `docker load`.
+
+However, v1 Image JSON (`application/vnd.docker.container.image.v1+json`) has been still widely
+used and officially adopted in [V2 manifest](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md)
+and in [OCI Image Format Specification](https://github.com/opencontainers/image-spec), which is
+the current industry's standard image specification. 
+
+## v1.X rough Changelog
+
+All 1.X versions are compatible with older ones.
+
+### [v1.2](v1.2.md)
+
+* Implemented in Docker v1.12 (July, 2016)
+* The official spec document was written in August 2016 ([#25750](https://github.com/moby/moby/pull/25750))
+
+Changes:
+
+* `Healthcheck` struct was added to Image JSON
+
+### [v1.1](v1.1.md)
+
+* Implemented in Docker v1.10 (February, 2016)
+* The official spec document was written in April 2016 ([#22264](https://github.com/moby/moby/pull/22264))
+
+Changes:
+
+* IDs were made into SHA256 digest values rather than random values
+* Layer directory names were made into deterministic values rather than random ID values
```

Comment:
While this impacts the specification, this was a change in the implementation. Should this be included in a historical account of the specification? May have an impact on docker section for each version. For example, 1.9 required migration and 1.10 included these changes.

Reply:
Could you elaborate what change is needed?
Do you mean v1.9 made names deterministic but did not introduce the new manifest JSON?